app/components/ExternalLinksBlock/components/ExternalLink/ExternalLink.jsx

```diff
@@ -32,6 +25,22 @@ export default function ExternalLink({ className, isExternalLink, isSocialLink,
     );
   }
 
+  if (isSocialLink) {
+    const icon = socialIcons[site].icon;
+    iconClasses = `fa ${icon} fa-fw`;
+    linkLabel = path;
+    linkProps['aria-label'] = socialIcons[site].ariaLabel;
+    const isValidLink = !!icon && linkProps.href.substring(0, 8) === 'https://';
```

Comment:
[minor] An alternate to `linkProps.href.substring(0, 8)` is `/^https?:\/\//i.test(linkProps.href)`, if anyone thinks RegEx is easier to read and/or if we need to consider Social Links that use `http://`

Reply:
I'm inclined to build the link from `link.site` and `link.path`, rather than use `link.url`.